native/pca_bench.cpp

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright (C) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <vector>
+#include <utility>
+#include <algorithm>
+#include <iostream>
+#include <chrono>  
+
+#include "common.hpp"
+#include "daal.h"
+
+
+namespace dn = daal::algorithms::normalization;
+
+
+std::pair<da::pca::ResultPtr, dm::NumericTablePtr>
+pca_fit_daal(double *X, size_t rows, size_t cols, size_t n_components) {
```

Comment:
It's good for now, but in the future, it might make sense to make this templated and run benchmarks for doubles as well as floats.

Reply:
Ok. We can basically do this for all benchmarks.